### PR TITLE
bulk upload entity

### PR DESCRIPTION
**Pull request by srikanthgudipudi**

### CODE REVIEW

File: src/app/modules/entities/bulk-entity-mapping/bulk-entity-mapping.component.html [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
-</p>
+<mat-toolbar role="toolbar" class="task-header">
+  <div class="title">Bulk Upload Entity Mapping</div>
+</mat-toolbar>
+<form [formGroup]="myForm" novalidate>
+  <mat-dialog-content class="mat-typography">
+    <div class="row" style="padding-left: 15px;">
+
+      <div>
+        <div class="space">
```
Comment (kiranharidas187) [inline src/app/modules/entities/bulk-entity-mapping/bulk-entity-mapping.component.html:9]: We planned to remove this right?

File: src/app/modules/entities/bulk-upload-entities/bulk-upload-entities.component.ts [hunk 2]
```diff
@@ -1,22 +1,68 @@
...
-  constructor(public fb: FormBuilder) { }
+  constructor(public fb: FormBuilder, private commonServiceService: CommonServiceService,
+    @Optional() @Inject(MAT_DIALOG_DATA) public data,
+    private dialog: MatDialog, private entityService: EntityService,
+    public dialogRef: MatDialogRef<BulkUploadEntitiesComponent>) { }
 
   ngOnInit() {
     this.reactiveForm();
+    this.getStateList();
+    this.getEntityTpes();
```
Comment (kiranharidas187) [inline src/app/modules/entities/bulk-upload-entities/bulk-upload-entities.component.ts:41]: Why are we calling on init?

File: src/app/modules/entities/entities-routing.module.ts [hunk 3]
```diff
@@ -12,13 +12,13 @@ const routes: Routes = [
       {
         path: 'list', component: StateEntityListComponent, data: {
           title: [{ name: 'Home', link: '/home' },
-          { name: 'StateEntity', link: 'entities/list' }
+          { name: 'StateListing', link: 'entities/list' }
```
Comment (kiranharidas187) [inline src/app/modules/entities/entities-routing.module.ts]: Put as **State Listing** or **States**

File: src/app/modules/entities/bulk-entity-mapping/bulk-entity-mapping.component.html (hunk 1 above)
Comment (srikanthgudipudi) [reply src/app/modules/entities/bulk-entity-mapping/bulk-entity-mapping.component.html:9]: No, This is bulk upload entity mapping modal.

**Pull request merged**